openedx/core/djangoapps/content_staging/filters.py

```diff
@@ -0,0 +1,55 @@
+"""
+Filters that affect the behavior of staged content (and the clipboard)
+"""
+# pylint: disable=unused-argument
+from __future__ import annotations
+
+from attrs import asdict
+
+from openedx_filters import PipelineStep
+from openedx_filters.tooling import OpenEdxPublicFilter
+from .data import StagedContentFileData
+from .models import StagedContent
+
+
+class StagingStaticAssetFilter(OpenEdxPublicFilter):
+    """
+    A filter used to determine which static assets associate with an XBlock(s)
+    should be staged in the StagedContent app (e.g. the clipboard).
+
+    This API is considered BETA. Once it is stable, this definition should be moved into openedx_filters.
+    """
+
+    filter_type = "org.openedx.content_authoring.staged_content.static_filter_source.v1"
```

Comment:
Not blocking, but I'm curious why this was done as a filter, as opposed to some new optional method on XBlocks?

Reply:
On second thought, now that I'm not using filters for python_lib.zip I think I'll do a follow-up PR to just remove this filter. YAGNI.